src/app/modules/main-module/components/main/main.component.ts

```diff
@@ -110,35 +129,46 @@ export class MainComponent implements OnInit {
     if (this.word[this.positionInput] !== '') {
       return;
     }
-
     if (this.positionInput > this.word.length - 1 || this.positionInput < 0) {
       this.positionInput = this.word.length - 1;
-
       return;
     }
-
     if (this.positionInput > 0) {
       this.positionInput--;
-
       return;
     }
   }
 
   deleteLetter() {
     this.changePositionWhenDelete();
-
     this.word[this.positionInput] = '';
   }
 
   setValuesWord() {
     this.wordSend.pos0 = this.word[0];
-
     this.wordSend.pos1 = this.word[1];
-
     this.wordSend.pos2 = this.word[2];
-
     this.wordSend.pos3 = this.word[3];
-
     this.wordSend.pos4 = this.word[4];
   }
+
+  checkWin() {
+    let resultado = this.wordStatus.join(',');
```

Comment:
en lugar de machear con un string así unido no podéis comprobarlo mejor por los códigos que en principio no cambiará? siempre mejor con identificadores las comparaciones de este tipo que con literales...